<def f='include/c++/5/bits/exception_ptr.h' l='99' ll='101' type='void std::__exception_ptr::exception_ptr::exception_ptr(std::__exception_ptr::exception_ptr &amp;&amp; __o)'/>
<use f='include/c++/5/bits/exception_ptr.h' l='118' u='c' c='_ZNSt15__exception_ptr13exception_ptraSEOS0_'/>
<use f='include/c++/5/bits/nested_exception.h' l='60' u='c' c='_ZNSt16nested_exceptionC1Ev'/>
